query/src/test/scala/hmda/query/repository/filing/FilingComponentSpec.scala

```diff
@@ -125,6 +123,24 @@ class FilingComponentSpec extends AsyncWordSpec with MustMatchers with FilingCom
 
     }
 
+    "Stream IRS" in {
```

Comment:
Sure thing, but that should be in a separate ticket, since that's behavior that's not affected by this pull request